### PR TITLE
FIX gulpfile for ES5 files

### PR DESCRIPTION
**Pull request by york-xtrem**
## All the info here

https://github.com/zurb/foundation-sites/issues/9302

### CODE REVIEW

**Comment by shafi49** [comment]
I highly recommend this merge request. I also faced similar problems while trying to use [ScrollMagic](http://scrollmagic.io/). 
However I somehow got it working by ignoring those files in .babelrc file like this 
image

`{
	"presets": ["es2015-without-strict"],
	"ignore": [
	"bower_components/justgage/raphael-2.1.4.min.js",
	"bower_components/scrollmagic/scrollmagic/uncompressed/ScrollMagic.js",
	"bower_components/scrollmagic/scrollmagic/minified/plugins/animation.gsap.min.js",
	"bower_components/scrollmagic/scrollmagic/minified/plugins/debug.addIndicators.min.js"
	],
  "compact": false
}`
In above code snippets I used "es2015-without-strict" to use [raphael.js](http://dmitrybaranovskiy.github.io/raphael/) library. As it turned out [raphael.js](http://dmitrybaranovskiy.github.io/raphael/) can not work while "[use strict](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode)" is enabled
Please consider this merge request so that people like me won't have to waste time trying to figure what went wrong.

**Comment by DanielRuf** [comment]
Please resolve the conflicts.

File: config.yml [hunk 1]
```diff
@@ -27,9 +27,9 @@ PATHS:
   sass:
     - "bower_components/foundation-sites/scss"
     - "bower_components/motion-ui/src"
-  # Paths to JavaScript libraries, which are combined into one file
-  javascript:
-    # Libraries required by Foundation
+  # Paths to JavaScript libraries, which are compined into one file
+  javascriptES6:
+    # Libraries requried by Foundation
```
Comment (DanielRuf) [inline config.yml]:
```suggestion
    # Libraries required by Foundation
```

File: config.yml [hunk 2]
```diff
@@ -27,9 +27,9 @@ PATHS:
   sass:
     - "bower_components/foundation-sites/scss"
     - "bower_components/motion-ui/src"
-  # Paths to JavaScript libraries, which are combined into one file
-  javascript:
-    # Libraries required by Foundation
+  # Paths to JavaScript libraries, which are compined into one file
```
Comment (DanielRuf) [inline config.yml]:
```suggestion
  # Paths to JavaScript libraries, which are combined into one file
```

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Pull request closed without merging**